db/migrate/20210204171358_update_invalid_telephone_numbers.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateInvalidTelephoneNumbers < ActiveRecord::Migration[6.1]
```

Comment:
I feel like I'd prefer this as script rather than a migration since nothing about the schema is changing.

Reply:
Sounds good, I like the idea of a script too, because then we can do cleanup via the script ahead of time, and be 100% sure we aren't invalidating any records in an important way.

Reply:
I've converted it to a script, and here it is as a [gist](https://gist.github.com/ngfreiter/bffa5bec6870797fb5245bf7b7b90b44#file-clean_phone_numbers-rb). My understanding is we would need to schedule another console session to run this, but let me know if that is not the case.

Reply:
Thanks! And yeah we would likely run the night of the release. @holmesie for awareness.

Reply:
If it's not too much of a pain, I would support trying to fix the invalid values before the release. That way we could run the script, see how many values are going to change, change them if it seems appropriate, and then double check that all of the records are looking valid. My concern is that if we do it on the night of, if something goes wrong then we may be in a situation where we want to back this PR out of the release on the fly. Thoughts?

Reply:
Oh agreed, we'd want to run this script (without modifying anything) way beforehand to at least get a sample and count of monitorees with invalid phone numbers.